reporters/jest/TerraVerboseReporter.js

```
@@ -0,0 +1,141 @@
+  }
+
+  log(message) {
+    const readableMessage = `${stripAnsi(message)}${endOfLine}`;
+    if (!this.isMonoRepo) {
+      if (readableMessage.search('\n') !== -1) {
+        this.results.output.push(readableMessage.split(/\n/g).forEach((piece) => {
+          this.unformattedResult.push(piece);
+        }));
+      }
+      this.results.output = this.unformattedResult.filter((obj) => obj);
+    }
+    if (this.isMonoRepo) {
+      this.setTestModule(readableMessage);
+      if (!this.results.output[this.moduleName]) {
+        this.results.output[this.moduleName] = [];
+      }
+      if (!this.unformattedResult[this.moduleName]) {
+        this.unformattedResult[this.moduleName] = [];
+      }
+      if (readableMessage.search('\n') !== -1) {
+        this.results.output[this.moduleName].push(readableMessage.split(/\n/g).forEach((piece) => {
+          this.unformattedResult[this.moduleName].push(piece);
+        }));
+      }
+      this.results.output[this.moduleName] = this.unformattedResult[this.moduleName].filter((obj) => obj);
+    }
+  }
+
+  onRunComplete() {
+    this.results.endDate = new Date().toLocaleString();
+    if (!this.isMonoRepo) {
+      fs.writeFile(this.filePathLocation, `${JSON.stringify(this.results, null, 2)}`, { flag: 'w+' }, (err) => {
+        if (err) {
+          Logger.error(err.message, { context: LOG_CONTEXT });
+        }
+      });
+    }
+    if (this.isMonoRepo) {
+      const { startDate, endDate, output } = this.results;
+      const moduleKeys = Object.keys(output) || [];
+      if (output && moduleKeys.length) {
+        moduleKeys.forEach(key => {
+          const fileData = {
+            startDate,
+            output: output[key],
+            endDate,
+          };
+          fs.writeFileSync(`${this.resultDir}/${key}.json`, `${JSON.stringify(fileData, null, 2)}`, { flag: 'w+' }, (err) => {
+            if (err) {
+              throw err;
```

do we want to throw the error or log the error like on line 117?